client/src/components/atoms/Modal.module.scss

```diff
@@ -11,11 +11,11 @@
     left: 0;
     right: 0;
     bottom: 0;
-    background: rgba($brand-black, .7);
+    background: rgba($brand-black, 0.7);
     overflow-x: hidden;
     overflow-y: auto;
     -webkit-overflow-scrolling: touch;
-    animation: fadeIn .2s ease-out backwards;
+    animation: fadeIn 0.2s ease-out backwards;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)